recipes/xspec/build.sh

```diff
@@ -23,19 +23,22 @@ if [ -n "$XSPEC_PATCH" ]
 then
     cd ${XSPEC_MODELS_ONLY}/Xspec/src;
     curl -LO -z ${XSPEC_PATCH} http://heasarc.gsfc.nasa.gov/docs/xanadu/xspec/issues/archive/${XSPEC_PATCH};
-    curl -LO -z ${XSPEC_PATCH_INSTALLER} http://heasarc.gsfc.nasa.gov/docs/xanadu/xspec/issues/archive/${XSPEC_PATCH_INSTALLER};
+    curl -LO -z ${XSPEC_PATCH_INSTALLER} http://heasarc.gsfc.nasa.gov/docs/xanadu/xspec/issues/${XSPEC_PATCH_INSTALLER};
     tclsh ${XSPEC_PATCH_INSTALLER} -m -n;
     rm -rf XSFits;
     cd ${XSPEC_DIR};
 fi
 
+#Copy in the OGIPTable fix
```

Comment:
I haven't been following this too closely, but I believe the OGIPTable fix is *only* for 12.10.0 and should not be applied to 12.10.1. Am I right in thinking it's currently applied to 12.10.1?

Reply:
I did keep the the OGIPTable fix for 12.10.1. If this is no longer needed, I can remove it and re-build.

Reply:
Before rebuilding I would check the differences in the files. If there are no significant differences there is no need to rebuild.